src/js/view/components/LogViewerList.js

```diff
@@ -107,11 +107,11 @@ const LogViewerList = props => {
   }, [props.lines]);
 
   useEffect(() => {
-    // Calculating the correct amount of lines needed to fill the page in the logviewer
+    // Calculating the amount of lines needed to fill the page in the logviewer
     setNumberOfLinesToFillLogView(
       Math.round(props.containerHeight / characterDimensions.height)
     );
-  }, [props.containerHeight]);
+  }, [props.containerHeight, characterDimensions]);
```

Comment:
Why is this needed? Do we expect that character dimensions to change?

Reply:
The character dimensions update on resize as seen in this effect. I found it useful to trigger the recalculation of the amount of lines in the viewer then. But I could be wrong.
```
useEffect(() => {
    // Handler to update the dimensions when needed
    const handleResize = () => {
      setListDimensions({
        width: logViewerListContainerRef.current.offsetWidth,
        height: logViewerListContainerRef.current.offsetHeight
      });

      setCharacterDimensions({
        width: oneCharacterSizeRef.current.offsetWidth,
        height: oneCharacterSizeRef.current.offsetHeight
      });
    };
    handleResize();

    // Calls are throttled to once every 200 ms
    const debouncedResizeHandler = _.debounce(handleResize, 200);
    window.addEventListener('resize', debouncedResizeHandler);
    // Return cleanup function
    return () => {
      window.removeEventListener('resize', debouncedResizeHandler);
    };
  }, []);
```

Reply:
Ah, of course. I thought they never updated. Makes sense to recalculate lines to fill then, nice catch 👍